dispatcher/MongoConnect.py

```diff
@@ -137,11 +140,11 @@ def GetUpdate(self):
                                                                     sort=[('_id', -1)])
                     self.latest_status[detector]['controller'][host] = doc
         except Exception as e:
-            print(type(e), e)
-            return -1
+            self.log.error(f'Got error while getting update: {type(e)}: {e}')
```

Comment:
not sure if the mongo client logger option will be used here but it might cause another exception if there is some mongo snaufau going on, probably not most important.

Reply:
100% of the time when this throws an error, the database was responsible, and I don't think we wrap the logger's db call in a try-except (todo: fix).